crates/net/rpc-api/src/admin.rs

```diff
@@ -33,4 +33,8 @@ pub trait AdminApi {
         item = String
     )]
     fn subscribe(&self);
+
+    /// Returns the ENR of the node.
```

Comment:
please have a look at:
https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-admin

we should return the same object here

sowe we need:
1. NodeInfo struct in rpc-types
2. populate this in `node_info`

Reply:
ahh my bad! should've asked for more context before just jumping into the issue. Will do so! also, the quicktype utility is super neat!